test/clt-tests/core/test-freeze-unfreeze-update.rec

```diff
@@ -75,3 +74,31 @@ mysql -P9315 -h0 -e "DESCRIBE t"
 mysql -P9315 -h0 -e "UPDATE t SET s='с' WHERE id=1; FLUSH RAMCHUNK t"; echo $?
 ––– output –––
 0
+––– input –––
+(sleep 1; mysql -P9315 -h0 -e "drop table if exists t; create table t(a string); insert into t values(1, 'a'); flush ramchunk t; freeze t"; mysql -P9315 -h0 -e "update t set a='b' where id = 1" 2>&1 &)
```

Comment:
You don't control that the 10K * 10 updates have completed by this moment. Is it on purpose?

Reply:
I changed the logic for this part of the test as well.